pkg/database/postgresql_database.go

```diff
@@ -0,0 +1,51 @@
+// Copyright KubeArchive Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package database
+
+import (
+	"fmt"
+
+	_ "github.com/lib/pq"
+)
+
+func init() {
+	RegisteredDatabases["postgresql"] = NewPostgreSQLDatabase
+}
+
+type PostgreSQLDatabaseInfo struct {
+	*DatabaseInfo
```

Comment:
Now that you are passing the environment to the constructor, I don't think that there is a need for having `DatabaseInfo` inside `PostgreSQLDatabaseInfo` I think that we can just have:
```suggestion
	env map[string]string
```

Reply:
But then new implementors will always have to add that `env`.  It seems cleaner to me to have the struct there and hide the implementation from the user.  No?

Reply:
And if `DatabaseInfo` were to change, it would require changes to all the implementers as well.

Reply:
> But then new implementors will always have to add that env

And now they have to add `DatabaseInfo` how is one thing different to another?
Also, if at some point someone wouldn't need the environment variables there is no need to add it.

> And if DatabaseInfo were to change, it would require changes to all the implementers as well.

I am not sure I'm getting this, if I am suggesting removing this object, how is this something that should be maintained?

Reply:
Never mind. I've fixed this.